src/Surfnet/StepupMiddleware/CommandHandlingBundle/Identity/Service/SecondFactorMailService.php

```diff
@@ -244,4 +268,28 @@ public function sendVettedEmail(
 
         $this->mailer->send($message);
     }
+
+    /**
+     * @param string $name
+     * @param string $locale
+     * @param string $fallbackLocale
+     * @return null|\Surfnet\StepupMiddleware\CommandHandlingBundle\Configuration\Dto\EmailTemplate
+     */
+    private function findEmailTemplate($name, $locale, $fallbackLocale)
+    {
+        $emailTemplate = $this->emailTemplateService->findByName($name, $locale, $this->fallbackLocale);
+
+        if ($emailTemplate) {
+            return $emailTemplate;
+        }
+
+        if ($this->warnOnMissingTemplateConfiguration) {
+            $this->logger->warning(
+                'Skipping sending mail because template configuration is missing',
```

Comment:
@rjkip but it's included in the context, why would it also need to be in the message?